Dockerfile

```diff
@@ -0,0 +1,15 @@
+# ---- Base image - order important ----
+FROM hmcts/ccd-definition-processor:latest as base
+
+# ----        Runtime image         ----
+FROM hmcts/ccd-definition-importer:latest as runtime
```

Comment:
These two base repos contain some basic scripts for uploading definitions etc, which we're not using at the moment but will be to automate the CCD config upload per PR

Reply:
`hmcts` wasn't it recently renamed?

Reply:
I _think_ they haven't published those images onto hmctspublic - or at least I cannot find them (tried `docker pull hmctspublic.azurecr.io/ccd-definition-importer:latest` for example but not luck) - I don't know if there is a way to list all the repos in that registry